17-react-native/classworks/Emelyanov_Egor/task/screens/Favourites/Favourites.screen.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { Text, View, TouchableOpacity, SafeAreaView, ScrollView, Image } from 'react-native';
+import { useNavigation } from '@react-navigation/native';
+import { useSelector } from 'react-redux';
+
+import styles from './Favourites.styles';
+import activeHeart from '../../assets/active_heart.png';
+import marvelLogo from '../../assets/logo.png';
+import { getFavouriteCharacters } from '../../resources/characters/characters.selectors';
+
+function HomeScreen() {
+  const navigation = useNavigation();
+
+  const favouritesCharacters = useSelector(getFavouriteCharacters);
+
+  return (
+    <SafeAreaView style={styles.container}>
+      <ScrollView>
+        <View style={styles.container}>
+          <Image
+            style={styles.logo}
+            source={marvelLogo}
+          />
+          <View>
+            <Text style={styles.title}>FAVOURITE CHARACTERS</Text>
+            {favouritesCharacters
+            &&
+            <View>
+            {favouritesCharacters.map(({ id, thumbnail, name, isFavourite }) => (
```

Comment:
i told you to make a CharactersList component and reuse it instead of rewriting the same code twice